connector/kafka-0-10-sql/src/test/resources/error/kafka-error-classes.json

```diff
@@ -0,0 +1,26 @@
+{
+  "TOPIC_PARTITIONS_IN_END_OFFSET_ARE_NOT_SAME_WITH_PREFETCHED" : {
+    "message" : [
+      "Kafka data source in Trigger.AvailableNow should provide the same topic partitions in pre-fetched offset to end offset for each microbatch. ",
+      "topic-partitions for pre-fetched offset: <tpsForPrefetched>, topic-partitions for end offset: <tpsForEndOffset>."
+    ]
+  },
+  "END_OFFSET_HAS_GREATER_OFFSET_FOR_TOPIC_PARTITION_THAN_PREFETCHED" : {
+    "message" : [
+      "For Kafka data source with Trigger.AvailableNow, end offset should have lower or equal offset per each topic partition than pre-fetched offset.",
+      "pre-fetched offset: <prefetchedOffset>, end offset: <endOffset>."
+    ]
+  },
+  "LOST_TOPIC_PARTITIONS_IN_END_OFFSET_WITH_TRIGGER_AVAILABLENOW" : {
+    "message" : [
+      "Some of partitions in Kafka topic(s) have been lost during running query with Trigger.AvailableNow. Make sure topic partitions are not dropped during the query run.",
```

Comment:
Same here.  Transient error right?

Reply:
Yup, maybe it's not a trivial case that users drop (and recreate) a topic which is being read from the running query. Better guidance may be just saying restart your query and it will work well.